src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -89,6 +90,13 @@ protected ReadThreadingAssemblerArgumentCollection getReadThreadingAssemblerArgu
             GVCFGQBands.add(70); GVCFGQBands.add(80); GVCFGQBands.add(90); GVCFGQBands.add(99);
     };
 
+    /**
+     * Output the band lower bound for each GQ block instead of the min GQ -- for better compression
+     */
+    @Advanced
+    @Argument(fullName=HaplotypeCallerArgumentCollection.OUTPUT_BLOCK_LOWER_BOUNDS, doc = "Output the band lower bound for each GQ block regardless of the data it represents", optional = true)
+    public boolean floorBlocks = false;
```

Comment:
Is there a test covering this new HC arg?

Reply:
Just in ReblockGVCF.  Do you want one in HaplotypeCaller?